src/main/kotlin/nl/avisi/structurizr/site/generatr/GenerateSiteCommand.kt

```diff
@@ -54,6 +54,15 @@ class GenerateSiteCommand : Subcommand(
         "Directory where the generated site will be stored. Will be created if it doesn't exist yet."
     ).default("build/site")
 
+    private val allBranches by option(
+        ArgType.Boolean, "allBranches", "all",
+        "When set to TRUE will generate a site for every branch in the git repository"
+    ).default(value = false)
+    private val excludeBranches by option(
+        ArgType.String, "excludeBranches", "ex",
+        "Comma-separated list of branches to exclude from the generated site"
+    ).default("gh-pages")
```

Comment:
The naming convention for program arguments is `kebab-case`, so please rename these two options to `all-branches` and `exclude-branches`.